src/utils/request.ts

```diff
@@ -1,44 +1,36 @@
-import axios from "axios";
 import { getUiUrl } from "./environmentVariables";
 import { reportError } from "./errorReporting";
 
 type optionsType = {
-  onFailure?: (e) => void;
+  onFailure?: (e: Error) => void;
```

Comment:
Do we need to support this option? The one place that calls this function doesn't use it, and I don't imagine we will use this function in other places

```suggestion
```